packages/lesswrong/server/styleGeneration.tsx

```diff
@@ -51,8 +52,9 @@ const generateMergedStylesheet = (themeOptions: ThemeOptions): Buffer => {
     jssStylesheet,
     ...theme.rawCSS,
   ].join("\n");
-  
-  return Buffer.from(mergedCSS, "utf8");
+
+  const minifiedCSS = minify(mergedCSS).css;
+  return Buffer.from(minifiedCSS, "utf8");
```

Comment:
Wait, this doesn't seem to have any prod checking. We **definitely** don't want to do this on dev.